packages/firebase_database/firebase_database/macos/firebase_database/Package.swift

```diff
@@ -0,0 +1,117 @@
+// swift-tools-version: 5.9
```

Comment:
Just noticed this, is it fairly recent or not? Just wondering

Reply:
I'm following the template in Flutter docs: https://docs.flutter.dev/packages-and-plugins/swift-package-manager/for-plugin-authors